testutils/testutils.go

```diff
@@ -0,0 +1,60 @@
+package testutils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// Output directory for testing and benchmark artifacts
+func GetTestOutDir() string {
+	out, _ := exec.Command("git", "rev-parse", "--show-toplevel").CombinedOutput()
+	repoRoot := string(out)
+	prefix := filepath.Join(strings.TrimSpace(repoRoot), "output")
+	return prefix
+}
+
+var (
+	ArchivesDir     = filepath.Join("test-artifacts", "archives")
```

Comment:
Shouldn't this be under `output` now? (i.e. `output/test-artifacts/{archives,oci-bundles,runs}`?